### DISCUSSION TITLE
Anyone interested in a Discord server?

### DISCUSSION THREAD

**Discussion by Sonosus** [FundamentalFrequency/.github#5] [General]
Title says it all really. I'm super excited about this project and think it would be a good idea to get a proper community going to support it. I'm happy to set one up if it's wanted.
EDIT: https://discord.gg/MatJsb67qb unofficial server!

**Comment by stonepreston** [comment]
I’d prefer to keep everything here in the GitHub discussions so all the project files, code, and any communication regarding the project is all kept in one spot

**Comment by tonn333** [comment]
Matrix based chat would be more fitting for this open source project.
https://matrix.org/clients/

**Comment by PaulREE** [comment]
> Title says it all really. I'm super excited about this project and think it would be a good idea to get a proper community going to support it. I'm happy to set one up if it's wanted.
> EDIT: https://discord.gg/MatJsb67qb unofficial server!

Can't join...

**Reply by Sonosus** [reply]
try this one... https://discord.gg/2tt3Rrk7fz

**Reply by Sonosus** [reply]
ping @PaulREE ^^
